docs/reference/cluster/stats.asciidoc

```diff
@@ -338,7 +348,289 @@ This object is not populated by the cluster stats API.
+nodes.
 
+`nodes.jvm.versions.version`:::
+(string)
+Version of JVM used by one or more selected nodes.
+
+`nodes.jvm.versions.vm_name`:::
+(string)
+Name of the JVM.
+
+`nodes.jvm.versions.vm_version`:::
+(string)
+Full version number of JVM.
++
+The full version number includes a plus sign (`+`) followed by the build number.
+
+`nodes.jvm.versions.vm_vendor`:::
+(string)
+Vendor of the JVM.
+
+`nodes.jvm.versions.bundled_jdk`:::
+(boolean)
+If `true`, the JVM includes a bundled Java Development Kit (JDK).
+
+`nodes.jvm.versions.using_bundled_jdk`:::
+(boolean)
+If `true`, a bundled JDK is in use by JVM.
+
+`nodes.jvm.versions.count`:::
+(integer)
+Total number of selected nodes using JVM.
+
+`nodes.jvm.mem.heap_used`::
+(<<byte-units, byte units>>)
+Memory currently in use by the heap across all selected nodes.
+
+`nodes.jvm.mem.heap_used_in_bytes`::
+(integer)
+Memory, in bytes, currently in use by the heap across all selected nodes.
+
+`nodes.jvm.mem.heap_max`::
+(<<byte-units, byte units>>)
+Maximum amount of memory, in bytes, available for use by the heap across all
+selected nodes.
+
+`nodes.jvm.mem.heap_max_in_bytes`::
+(integer)
+Maximum amount of memory, in bytes, available for use by the heap across all
+selected nodes.
+
+`nodes.jvm.threads`::
+(integer)
+Number of active threads in use by JVM across all selected nodes.
+
+`nodes.fs.total`::
+(<<byte-units, byte units>>)
+Total size of all file stores across all selected nodes.
+
+`nodes.fs.total_in_bytes`::
+(integer)
+Total size, in bytes, of all file stores across all seleced nodes.
+
+`nodes.fs.free`::
+(<<byte-units, byte units>>)
+Amount of unallocated disk space in file stores across all selected nodes.
+
+`nodes.fs.free_in_bytes`::
+(integer)
+Total number of unallocated bytes in file stores across all selected nodes.
+
+`nodes.fs.available`::
+(<<byte-units, byte units>>)
+Total amount of disk space available to JVM in file
+stores across all selected nodes.
++
+Depending on OS or process-level restrictions, this amount may be less than
+`nodes.fs.free`. This is the actual amount of free disk space the selected {es}
+nodes can use.
+
+`nodes.fs.available_in_bytes`::
+(integer)
+Total number of bytes available to JVM in file stores
+across all selected nodes.
++
+Depending on OS or process-level restrictions, this number may be less than
+`nodes.fs.free_in_byes`. This is the actual amount of free disk space the
+selected {es} nodes can use.
+
+`nodes.plugins`::
+(array of objects)
+Contains details about installed plugins and modules across all selected nodes.
++
+If no plugins or modules are installed, this array is empty.
+
+`nodes.plugins.name`:::
+(string)
+Name of the {es} plugin.
+
+`nodes.plugins.version`:::
+(string)
+{es} version for which the plugin was built.
+
+`nodes.plugins.description`:::
+(string)
+Short description of the plugin.
+
+`nodes.plugins.classname`:::
+(string)
+Class name used as the plugin's entry point.
+
```

Comment:
Missing `nodes.plugins.extended_plugins` typed `array[string]` which defines any other plugins this plugin extends through SPI, or empty array if it does not extend anything.

Reply:
Added this with 7d6a5e1cfa3dc301373ba3753a31d9ca20cafcef.